docs/CHANGELOG.asciidoc

```diff
@@ -51,6 +52,222 @@ Review important information about the {kib} 8.x releases.
+* Adds filter details to panel settings ({kibana-pull}162913[#162913]).
+Discover::
+* Redesign for the grid, panels and sidebar ({kibana-pull}165866[#165866]).
+* Set data table row height to auto-fit by default ({kibana-pull}164218[#164218]).
+* Allow fetching more documents on Discover page ({kibana-pull}163784[#163784]).
+* Fixes unified search for long field inputs ({kibana-pull}166024[#166024]).
+Elastic Security::
+For the Elastic Security 8.11.0 release information, refer to {security-guide}/release-notes.html[_Elastic Security Solution Release Notes_].
+Enterprise Search::
+For the Elastic Enterprise Search 8.11.0 release information, refer to {enterprise-search-ref}/changelog.html[_Elastic Enterprise Search Documentation Release notes_].
+Fleet::
+* Adds sidebar navigation showing headings extracted from the readme ({kibana-pull}167216[#167216]).
+Lens & Visualizations::
+* Other bucket defaults to false for top values greater than equal 1000 in *Lens* ({kibana-pull}167141[#167141]).
+* Adds support for decimals in percentiles in *Lens* ({kibana-pull}165703[#165703]).
+* Support date fields in options list ({kibana-pull}164362[#164362]).
+Machine Learning::
+* Update ELSER version for Elastic Assistant ({kibana-pull}167522[#167522]).
+* Retain `created_by` setting when exporting anomaly detection jobs ({kibana-pull}167319[#167319]).
+* Improved wording of awaiting ML nodes messages ({kibana-pull}167306[#167306]).
+* Adding `created_by` job property for the advanced wizard ({kibana-pull}167021[#167021]).
+* Trained model testing: only show indices with supported fields ({kibana-pull}166490[#166490]).
+* Alerts as data integration for Anomaly Detection rule type ({kibana-pull}166349[#166349]).
+* Data Frame Analytics Trained models: add ability to reindex after pipeline creation success ({kibana-pull}166312[#166312]).
+* Improvements to UX of adding ML embeddables to a dashboard ({kibana-pull}165714[#165714]).
+* AIOps: Support text fields in log rate analysis ({kibana-pull}165124[#165124]).
+* Data Frame Analytics creation wizard: adds ability to add custom URLs to jobs ({kibana-pull}164520[#164520]).
+Management::
+* Adds Create a data view button to index or saved search selector in ML pages and Transforms management ({kibana-pull}166668[#166668]).
+* Improve loading behavior of Transform ({kibana-pull}166320[#166320]).
+* Clusters tab added under Inspector ({kibana-pull}166025[#166025]).
+* Index details can now be viewed on a new index details page in Index Management ({kibana-pull}165705[#165705]).
+* Adds support for PATCH requests in Console ({kibana-pull}165634[#165634]).
+* Improves autocomplete to suggest knn in search query ({kibana-pull}165531[#165531]).
+* Improves display for long descriptions in Transforms ({kibana-pull}165149[#165149]).
+* Improve transform list reloading behavior in Transforms ({kibana-pull}164296[#164296]).
+Observability::
+* ES|QL query generation ({kibana-pull}166041[#166041]).
+Presentation::
```

Comment:
If you put both line items relating to the inspector in their own heading, you could rename this one to `Maps` potentially.